cmd/minikube/cmd/start_flags.go

```diff
@@ -221,6 +227,12 @@ func initNetworkingFlags() {
 	startCmd.Flags().String(serviceCIDR, constants.DefaultServiceCIDR, "The CIDR to be used for service cluster IPs.")
 	startCmd.Flags().StringArrayVar(&config.DockerEnv, "docker-env", nil, "Environment variables to pass to the Docker daemon. (format: key=value)")
 	startCmd.Flags().StringArrayVar(&config.DockerOpt, "docker-opt", nil, "Specify arbitrary flags to pass to the Docker daemon. (format: key=value)")
+
+	// generic
+	startCmd.Flags().String(genericIPAddress, "", "IP address (generic)")
+	startCmd.Flags().String(genericSSHUser, defaultSSHUser, "SSH user (generic)")
```

Comment:
lets add a Validation for these flags, so if users passes them to other drivers there should be exit.Usage
"generic-ssh-user: can only be used by generic driver...."

Reply:
also a doc page in the website would be nice.

Reply:
The other ones say "xxx driver only", so will go with that